addons/mail/static/src/models/thread/thread.js

```diff
@@ -419,9 +419,8 @@ function factory(dependencies) {
                 orderBy: [{ name: 'id', asc: false }],
             }));
             for (const attachmentData of attachmentsData) {
-                this.env.models['mail.attachment'].insert(Object.assign({
-                    originThread: [['link', this]],
-                }, this.env.models['mail.attachment'].convertData(attachmentData)));
+                this.update({ originThreadAttachments: [['insert-and-replace', attachmentsData.map(
+                    attachmentData => this.env.models['mail.attachment'].convertData(attachmentData))]] })
```

Comment:
Would consider this as a bug-fix in saas-13.4.

Reply:
What I need to do ? Split the commit ?

Reply:
You can make a PR with single `[FIX]` commit that targets saas-13.4 with this change, including test. Thanks!